patt/src/main/java/org/example/builder/Computer.java

```diff
@@ -0,0 +1,26 @@
+package org.example.builder;
+
+public class Computer {
+    private String prossesor;
+    private RAM RAM;
```

Comment:
N: RAM/Ram it's okay for enum type naming, but name of variable should be in lowercase.

Ex:
Ram ram;
RAM ramType;
RAM ram;